packages/patrol_cli/lib/src/test_finder.dart

```diff
@@ -62,7 +62,7 @@ class TestFinder {
 
         testFiles.addAll(foundTargets);
       } else {
-        throwToolExit('target $target is invalid');
+        throwToolExit('target $target is invalid. Does your test file(s) end with "_test.dart"?');
```

Comment:
This error message would be incorrect in case when someone will set custom test file suffix using pubspec config. Could you update the error so that it will use the suffix?